server/server.js

```diff
@@ -69,15 +83,21 @@ var bedPath;
  */
 var mappingPath;
 /**
- * Path of user logs.
+ * Path of config file.
  * @type {string}
  */
-var logPath;
+var configPath = 'server/config';
+
 /**
  * Path of config file.
  * @type {string}
  */
-var configPath = 'server/config';
+var privateKeyPath = '/etc/ssl/private/star_simonsfoundation_org.key';
```

Comment:
Key path should be set in config file. The github repo shall not contain anything related to simons. You are exposing a critical file path from the foundation's server.